commit 6d917e872c5e94acfec16e36bbe56e8c7e242a4d
Author: Miles Alan <m@milesalan.com>
Date:   Tue Oct 13 21:54:04 2020 -0500

    Cast c->pageid to int in dbg message to allow build to work

diff --git a/surf.c b/surf.c
index b83a596..5c1c9c2 100644
--- a/surf.c
+++ b/surf.c
@@ -1930,7 +1930,7 @@ msgext(Client *c, char type, const Arg *a)
 	int ret;
 
 	if ((ret = snprintf(msg, sizeof(msg), "%c%c%c%c",
-	                    4, c->pageid, type, a->i))
+	                    4, (int) c->pageid, type, a->i))
 	    >= sizeof(msg)) {
 		fprintf(stderr, "surf: message too long: %d\n", ret);
 		return;
